### DISCUSSION TITLE
mediafire blocked

### DISCUSSION THREAD

**Discussion by m0nesix** [t1m0thyj/WinDynamicDesktop#556] [Q&A]
Hello, I have a problem. I'm from Kazakhstan and the site where you can download themes is blocked. I tried to find them on other sites but I couldn't where can I download them?

**Comment by t1m0thyj** [comment]
Hello, unfortunately there isn't any other location at this time where community themes for WinDD can be downloaded. Are you able to connect to the Internet with a VPN to download from Mediafire?
